### ISSUE TITLE
Performance issues

### ISSUE THREAD

**Issue by SangeethaS94** [openlayers/ol-cesium#551]
I have a vector layer with approximately 6000 polygons. In 2D view, all polygons are drawn within few seconds, however in 3D, it takes around 5 minutes. Is there a way to improve the performance? 

Also, does ol-cesium support **renderMode : 'image'** for vector layers? Is it possible to improve performance using this mode?

**Comment by gberaudo** [comment]
I guess your performance issue comes from the fact that OL-Cesium immediately instanciates your 6000 polygons to pass them to Cesium.

After instanciation is done, is Cesium navigation fluent?

A solution would be to instanciate them step by step during several animation frames. It would make sense to instanciate in priority the polygons in the viewport and add them to Cesium as they are ready.

No there is no support for `renderMode: 'image'`. It would require writing a special Cesium Imagery Provider that would take its tiles from the rendered images.

Both ideas above are interesting and contributions implementing them are welcome.

**Comment by SangeethaS94** [comment]
Yes the navigation is fluent after all the features are loaded. However, when I add another layer containing approximately 5000 polygons, the 3D view becomes blank, either black or white and never works again. 

Regarding the performance issue, I noticed that the polygons are drawn faster when the layer is not visible. Hence, hide the layer a second after it is added to the map and when I make it visible after about 30 seconds, all features are drawn.
